src/App.jsx

```diff
@@ -162,13 +160,9 @@ function App() {
 
             {/* ----- Canvas ------ */}
             <canvas
-              id="draw"
-              className={`whiteboard bg-slate-950 w-screen mt-[4vh] rounded-[0.6rem] shadow-md shadow-black dark:shadow-black dark:shadow-lg ${
-                isDrawing ? "cursor-crosshair" : "cursor-default pointer-events-none"
-              }
-            `}
-              ref={canvasRef}
-            ></canvas>
+              // id="draw"
+              className={`whiteboard bg-slate-950 rounded-[0.6rem] shadow-md shadow-black dark:shadow-black dark:shadow-lg cursor-crosshair`}
```

Comment:
You don't need to use backticks and curly brackets {``}.

React evaluates the expression inside the curly braces, even if it's just a static string, which involves a bit more processing.

It's not a best practice though, We only want to use backticks and curly braces when we need to include JavaScript expressions, such as conditional logic, variables, or template literals.

So, you need to use double quotes for static class names to avoid unnecessary processing.

Additionally you can remove the `id="draw"`, it's not needed.

You can copy paste the below suggestion,

```suggestion
className="whiteboard bg-slate-950 rounded-[0.6rem] shadow-md shadow-black dark:shadow-black dark:shadow-lg cursor-crosshair"
```

Reply:
I changed it now. Please check once, is it okay now...?